### PR TITLE
Add docs/recipes for Chrome Dev Editor

### PR DESCRIPTION
**Pull request by SmokyBob**
Implementation of #121 `docs\recipes` for CDE

### CODE REVIEW

**Comment by chuckh** [comment]
LGTM

File: docs/recipes/chrome-dev-editor.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Use Polymer Starter Kit on Chrome Dev Editor
+
+If you are using a Chromebook, one of the few IDE you can use is Chrome Dev Editor.
```
Comment (arthurvr) [inline docs/recipes/chrome-dev-editor.md]: Maybe link to the CDE website?

File: docs/recipes/chrome-dev-editor.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+To use the Polymer Starter Kit you have to download the [latest release](https://github.com/PolymerElements/polymer-starter-kit/releases) in the `light` flavor (the additional tools can't be used from CDE).
+
+After downloading the `polymer-starter0kit-light-*.zip` file unpack it in a new folder (for Example `psk-light`) you should have a directory structure like
```
Comment (arthurvr) [inline docs/recipes/chrome-dev-editor.md]: > polymer-starter0kit-light-*.zip

The 0 seems like a typo.

File: docs/recipes/chrome-dev-editor.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+If you are using a Chromebook, one of the few IDE you can use is Chrome Dev Editor.
+
+To use the Polymer Starter Kit you have to download the [latest release](https://github.com/PolymerElements/polymer-starter-kit/releases) in the `light` flavor (the additional tools can't be used from CDE).
+
+After downloading the `polymer-starter0kit-light-*.zip` file unpack it in a new folder (for Example `psk-light`) you should have a directory structure like 
+
+![psk-light-folder-p1](https://cloud.githubusercontent.com/assets/1431346/9451900/a73ffcf2-4ab1-11e5-8742-e0b5523ba9d5.png)
+
+
+Before opening the folder inside CDE, we need to move the file `bower.json` to `app/bower.json`, this way running `Bower Update` will place the updated packages in `app/bower_components`
```
Comment (arthurvr) [inline docs/recipes/chrome-dev-editor.md]: > `Bower Update`

`bower update`

File: docs/recipes/chrome-dev-editor.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+![bower json-post](https://cloud.githubusercontent.com/assets/1431346/9452119/c5826a46-4ab2-11e5-96c5-00cf404d9c50.png)
+
+
+We can now `Open Folder...` inside CDE and start renaming the file `app/manifes.json` to `app/web-app-manifest.json`, followed by updating the link to it in the file `app/index.html`
```
Comment (arthurvr) [inline docs/recipes/chrome-dev-editor.md]: > app/manifes.json

`manifest`

File: docs/recipes/chrome-dev-editor.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+page('*', function () {
+  app.route = 'home';
+});
+```
+
+After the code will look like the following
+
+```
+...
+    page('/contact', function () {
```
Comment (arthurvr) [inline docs/recipes/chrome-dev-editor.md]: The indentation at the begin here is kinda moot.

File: docs/recipes/chrome-dev-editor.md (hunk 3 above)
Comment (SmokyBob) [inline docs/recipes/chrome-dev-editor.md]: in CDE the menu it's called "Bower Update" that's why I've use the capital letters
image

Should I use the command line version or make more clear I'm referring to the menu?

File: docs/recipes/chrome-dev-editor.md (hunk 1 above)
Comment (SmokyBob) [inline docs/recipes/chrome-dev-editor.md]: Yep, I agree.
I'll add a link to the github repo of the project

File: docs/recipes/chrome-dev-editor.md (hunk 3 above)
Comment (arthurvr) [inline docs/recipes/chrome-dev-editor.md]: Oh, no, the capital letters seem alright, then.

**Comment by addyosmani** [comment]
@SmokyBob If you think we can nail the above nit Marc found (re: routing) and squash commits this should be all good to go :star:

**Comment by SmokyBob** [comment]
:+1:

**Comment by addyosmani** [comment]
Landing. Thanks @SmokyBob and all our lovely reviewers :heart_eyes_cat:

**Pull request merged**